### DISCUSSION TITLE
Is there a benchmark for `PIXI.Text` vs `PIXI.HTMLText`?

### DISCUSSION THREAD

**Discussion by UlyssesZh** [pixijs/pixijs#9667] [Q&A] [answered]
I heard that HTMLText is faster than Text, but when I tried to migrate the texts in my game to HTMLText, I found that it led to very serious FPS lag and made the game unplayable. I use an external font (I have already loaded the font by awaiting `HTMLTextStyle.loadFont` in the loading phase of that scene), and there is only one short but large font size (~1000) text in the scene.

I wonder whether there is a benchmark for Text vs HTMLText to let me see in what circumstances HTMLText is faster.

**Comment by bigtimebuddy** [comment, marked as answer]
Some context here: https://github.com/pixijs/html-text/issues/38

Custom fonts may be slower. We'll have to do some more testing to see.

**Reply by UlyssesZh** [reply]
![image](https://github.com/pixijs/pixijs/assets/26196187/d2c62245-c0ec-4200-b5e5-9a477e4e7aee)
Not just slower, but **extremely** slower... The number is just the time needed to update **once** (the original benchmark program was to 1000 of such measures). https://jsfiddle.net/UlyssesZhan/naxhmgd5/1/
Run on Chrome 116, Ubuntu 20.04, PixiJS 7.2.4.
